rfcs/0006-cdevents/README.md

```diff
@@ -0,0 +1,134 @@
+# RFC-0006 Flux CDEvents Receiver
+
+<!--
+The title must be short and descriptive.
+-->
+
+**Status:** provisional
+
+<!--
+Status represents the current state of the RFC.
+Must be one of `provisional`, `implementable`, `implemented`, `deferred`, `rejected`, `withdrawn`, or `replaced`.
+-->
+
+**Creation date:** 2023-12-08
+
+**Last update:** 2024-02-09
+
+## Summary
+
+The CDEvents Receiver for Flux receives a CDEvent sent to the receiver webhook URL, verifies it and checks the event type against `.spec.events`, then triggers reconciliation of the configured resources.
```

Comment:
This is just a styling comment, but could you start the Summary with an introduction sentence?